src/main/java/com/github/cameltooling/dap/internal/model/variables/message/MessageExchangeVariablesVariable.java

```diff
@@ -38,4 +59,25 @@ private Variable createVariable(String key, String value) {
 		return variable ;
 	}
 
+	public SetVariableResponse setVariableIfInScope(SetVariableArguments args, ManagedBacklogDebuggerMBean debugger) {
+		if (args.getVariablesReference() == getVariablesReference()) {
+			debugger.setExchangeVariableOnBreakpoint(breakpointId, args.getName(), args.getValue());
+			if (exchangeVariables != null) {
+				for (ExchangeVariable exchangeVariable : exchangeVariables) {
+					if (exchangeVariable.getKey().equals(args.getName())) {
+						exchangeVariable.setValue(args.getValue());
```

Comment:
I think we can call a break here (and also in the MessageExchangePropertiesVariable from where it was inspired)

Reply:
added `break`